### PR TITLE
Refresh button on home page

### PR DESCRIPTION
**Pull request by tinacho**

### CODE REVIEW

File: components/team/Styles.js [hunk 1]
```diff
@@ -46,4 +46,19 @@ const MemberItem = styled.li`
   margin-bottom: 5px;
 `;
 
-export { Box, Title, Form, Section, Strong, MemberItem };
+const StyledButton = styled.button`
```
Comment (JohannesMerz) [inline components/team/Styles.js:49]: bit odd to have this inside the team folder if its used by outside components but i will approve to not slow things down

**Review by JohannesMerz — APPROVED** [review]

File: components/team/Styles.js (hunk 1 above)
Comment (tinacho) [reply components/team/Styles.js:49]: I know, I didn't wanna move it out and then have to check every view 😄

Comment (tinacho) [reply components/team/Styles.js:49]: But now it's done, happy? 😛

Comment (JohannesMerz) [reply components/team/Styles.js:49]: So happy

**Pull request merged**